mathics/builtin/numbers/numbertheory.py

```diff
@@ -141,11 +143,54 @@ def apply(self, n, evaluation):
 #                'List', *(Integer(c) for c in coeff)))
 
 
+class EulerPhi(SympyFunction):
+    """
+        <dl>
+          <dt>'EulerPhi[$n$]'
+          <dd>returns the Euler totient function .
+        </dl>
+
+    EulerPhi is also known as the Euler totient function or phi function.
+    It is typically used in cryptography and in many applications in elementary number theory.
+
+    EulerPhi[n] counts positive integers up to n that are relatively prime to n.
+
+        Compute the Euler totient function:
+        >> EulerPhi[9]
+        = 6
+
+        'EulerPhi' of a negative integer is same as its positive counterpart:
+        >> EulerPhi[-11] == EulerPhi[11]
+        = True
+
+        Large arguments are computed quickly:
+        >> EulerPhi[40!]
+        = 121343746763281707274905415180804423680000000000
+
+        'EulerPhi' threads over lists:
+        >> EulerPhi[Range[1, 17, 2]]
+        = {1, 2, 4, 6, 6, 10, 12, 8, 16}
+        Above, we get consecutive even numbers when the input is prime.
+
+        Compare the results above with:
+        >> EulerPhi[Range[1, 17]]
+        = {1, 1, 2, 2, 4, 2, 6, 4, 6, 4, 10, 4, 12, 6, 8, 8, 16}
+    """
+
+    attributes = listable | numeric_function | protected
+    summary_text = "Euler totient function"
+    sympy_name = "totient"
+
+    def apply(self, n, evaluation):
+        "EulerPhi[n_Integer]"
+        return super().apply(abs(n), evaluation)
```

Comment:
Note that the addition of `abs()` in `Integer` of core.py which builds on a recent change simplifies this.